v22.1/information-schema.md

```diff
@@ -358,7 +358,7 @@ Column | Description
 `table_schema` | Name of the schema that the grant applies to.
 `table_name` | Name of the table that the grant applies to.
 `privilege_type` | Type of [privilege](security-reference/authorization.html#managing-privileges): `SELECT`, `INSERT`, `UPDATE`, `DELETE`, `TRUNCATE`, `REFERENCES`, or `TRIGGER`.
-`is_grantable` | Always `NULL` (unsupported by CockroachDB).
+`is_grantable` | <span class="version-tag">New in v22.1:</span> `TRUE` if the grantee has the admin option on the table; `FALSE` if not.
```

Comment:
it should say "has the grant option." that is different from the "admin option"

the grant option modifies what you can do with a SQL privilege like SELECT, UPDATE, INSERT, or USAGE

the admin option modifies what you can do when granting roles to users.